src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Serialization.cs

```diff
@@ -51,14 +46,14 @@ internal partial class SymbolTreeInfo : IObjectWritable
             Solution solution,
             Checksum checksum,
             string filePath,
-            Func<SpellChecker> create)
+            Func<Task<SpellChecker>> createAsync)
```

Comment:
❕ Use `Func<CancellationToken, Task<SpellChecker>>` so the scope of the token within the callback is easier to understand.